services/cadip/rs_server_cadip/cadip_utils.py

```diff
@@ -18,12 +18,50 @@
 It includes an API endpoint, utility functions, and initialization for accessing EODataAccessGateway.
 """
 
+import os
+import os.path as osp
+from pathlib import Path
 from typing import Dict, List
 
 import eodag
 import starlette.requests
+import yaml
 
 DEFAULT_GEOM = {"geometry": "POLYGON((180 -90, 180 90, -180 90, -180 -90, 180 -90))"}
+CADIP_CONFIG = Path(osp.realpath(osp.dirname(__file__))).parent / "config"
+search_yaml = CADIP_CONFIG / "cadip_search_config.yaml"
+
+
+def validate_cadip_config(fp):
+    """Function to validate yaml template, tba. Should we check this?"""
+    accepted_stations = ["cadip", "ins", "mts"]  # pylint: disable=unused-variable # noqa
```

Comment:
please don't hardcode a list of stations in the source code. One goal of the configuration file is exactly to avoid to change the software if we want to add new stations.

Reply:
Agree, this function will be removed, it's not used.